# Suraksha Vinod
**Email:** suraksha.vinlak@gmail.com \
**LinkedIn:** https://linkedin.com/in/surakshavinod \
**GitHub:** https://github.com/surakshavinod

![image](https://medlineplus.gov/images/PX000098_PRESENTATION.jpeg)

## Education
**Doctor of Philosophy** (Bioinformatics and Genomics) \
The Pennsylvania State University - State Collge, PA \
Aug '25 - Present

**Master of Science** (Bioinformatics) \
Georgia Institute of Technology - Atlanta, GA \
Aug '23 - May '25

**Bachelor of Technology** (Biotechnology) \
PES University - India \
Aug '19 - May '23


## Experience
**Graduate Research Assistant - Georgia Institute of Technology**
_Jan '24 - Dec '24_ \
Atlanta, GA 
- Developed a computational pipeline to investigate the influence of ancestry and weathering on the polygenic scores (PGS) in diverse whole genome sequencing (WGS) dataset, from creation of ancestry profile, phenotypic data cleaning and downstream processing of genotypic data to data visualization. 
- Constructed prevalence-risk curves across five types of diseases with respect to education and income, stratified by location to understand the risk trajectory.
- Applied statistical tests to understand the significance of the results.


**Summer Intern - M.D. Anderson Cancer Center**
_Jun '24 - Aug '24_ \
Houston, TX 
- Implemented machine learning models with over 35,000 features to predict the tumor-specific total mRNA expression (TmS) across cancer types, achieving over 0.8 CCC and low MSE.
- Performed survival analysis on TCGA datasets to examine time-to-event data, successfully identifying the significant factors influencing patient outcomes, which informed clinical decision-making.
- Conducted Gene Set Enrichment Analysis (GSEA) with over 30,000 genes to validate the results, identifying the significantly over-expressed genes and pathways associated with cancer, contributing to the understanding of disease mechanisms. 

**Project Intern - Indian Institute of Science**
_Jan '23 - Jun '23_ \
Bengaluru, India 
- Spearheaded genome wide association studies (GWAS) utilizing WGS data with emphasis on rare variant association in a cohort of over 2000 individuals. 
- Pioneered the development and optimization of an extensive parallelized computational pipeline to detect disease associations with rare variants and identify causal variants, and interpretation of the GWAS results using statistical plots. 
- Developed an automated computational pipeline to run quality control measures and downstream genomic data analysis in Python. 

**Research Intern - Leucine Rich Bio Pvt Ptd**
_Jun '22 - Dec '22_ \
Bengaluru, India 
- Conducted a thorough literature survey, compiling data on the gut microbiome and its association with neurodegenerative diseases, specifically Parkinson’s disease (PD), from multiple sources.
- Curated and managed a comprehensive database highlighting key insights, facilitating data interpretation.
- Explored the implications of gut microbiome on PD by carrying out Bioinformatics data analysis using Galaxy.

## Projects
1. **Alzheimer’s Disease classifier from fMRI data** \
Leveraged machine learning approaches to accurately classify and predict Alzheimer’s Disease patients, Mild Cognitive Impairment patients, and Healthy Controls using 60 fMRI scans from the ADNI database, achieving a prediction accuracy of 78%. 
2. **CityScout - Identifying Safe Cities Based on Crime Statistics** \
Lauched a web-based tool to identify safe U.S. cities for various use cases (e.g., girls’ trips, family relocation, vacation with children, etc.) and customizable filters tailored to specific demographics (e.g., resident, gender, children, etc.) by analyzing 2TB of FBI crime data using PySpark on Google Cloud Platform and Tableau for data visualization.
3. **Comparison of ML models to predict absorption properties** \
Applied and compared four machine learning models to predict human plasma protein binding rate (PPBR), bioavailability (BIO) and parallel artificial membrane permeability assay (PAMPA) aiming to contribute to the advancement of predictive modeling in drug discovery. Utilized at least 11 features per property and the PyTDC data to build models, achieving a model accuracy of 71%.


## Manuscripts and Presentations
1. Ankita Paul, Shuangxi Ji, Jessica C. Lal, Carissa Fong, **Suraksha Vinod**, Huiying Hu, Kevin Chen, Matthew Montierth, Quang Tran, Wenyi Wang. Machine learning models predict tumor-specific total mRNA expression associated with clinical outcomes across 14 cancer types [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1095.
2. **Vinod, S.**, Rajesh, S., KN, S., Karunakar, P. (2024). Identifying anti-diabetic potential of alpha-amylase inhibitors in _Ocimum basilicum_ using In vitro and In silico approaches. Suranaree Journal of Science and Technology, 30(6), 030159(1-10). https://doi.org/10.55766/sujst-2023-06-e01323
3. **Vinod, S.**, Karunakar, P. (Poster) Identification of potential inhibitor against Nipah virus using In silico approach. I-SENTIENT, School of Sciences, Jain (Deemed-to-be) University. September 2022.
4. **Vinod, S.**, Rajesh, S., Karunakar, P., KN, S. Evaluating the anti-diabetic activity of alpha-amylase inhibitors in _Cuminum cyminum_ using In vitro and In silico approach. International Journal of Computational Biology and Drug Design. Under review.

## Awards and Grants
- Computational Biology Faculty Research Award (x2) - 2024
- Prof. M R Doreswamy Merit Scholarship Award Certificate (x5) - 2020 - 2023
- Best Paper Award - I-SENTIENT 2022
- 1st Runners-up - Nutrathon 2022 
- Millennium Fellowship (by United Nations Academic Impact and Millennium Campus Network) 2021
- 2nd Runners-up - Druggathon 2021 2021
- Distinction Award Certificate (x2) 2020 - 2023

## Skills
 - **Programming/Workflow/Containers:** Python, R, SQL, D3.js, Nextflow, Docker
 - **Bioinformatics Software:** DESeq2, BLAST, Galaxy, AlphaFold, AutoDock Vina, Desmond, GSEA, PLINK, VCF/BCF Tools, Annovar, SwissADME, OpenBabel, LigPlot+, CellDesigner
 - **Libraries/Other:** Libraries/Other: Git, Bash, HPC, AWS, NumPy, Pandas, scikit-learn, Matplotlib, dplyr, tidyr, Seurat, ggplot2, survival, survminer, TCGAbiolinks, seaborn, shiny, conda/miniconda, Jupyter Notebook, OpenRefine, Python Flask
 - **Languages:** English, Hindi, Kannada, French (Beginner), German (Beginner)

