---
title: "Suraksha Vinod"
  links:
    - icon: envelope
      text: Email
      href: mailto:suraksha.vinlak@gmail.com
    - icon: linkedin
      text: LinkedIn
      href: https://linkedin.com/in/surakshavinod
    - icon: github
      text: GitHub
      href: https://github.com/surakshavinod
---

üìç State College, PA

---

## Education
**Doctor of Philosophy ‚Äì Bioinformatics and Genomics**  
*The Pennsylvania State University, State College, PA*  
Aug 2025 ‚Äì Present  

**Master of Science ‚Äì Bioinformatics**  
*Georgia Institute of Technology, Atlanta, GA*  
Aug 2023 ‚Äì May 2025  

**Bachelor of Technology ‚Äì Biotechnology**  
*PES University, India*  
Aug 2019 ‚Äì May 2023  

---

## Experience
**Graduate Research Assistant - Georgia Institute of Technology**  
_Jan '24 - Dec '24_  
Atlanta, GA  

- Developed a computational pipeline to investigate the influence of ancestry and environment on polygenic scores across five diseases in a diverse WGS cohort of ~10,000 individuals
- Integrated DNA methylation data to explore the impact of weathering on the disease risk profiles

**Summer Intern - M.D. Anderson Cancer Center**  
_Jun '24 - Aug '24_  
Houston, TX  

- Implemented machine learning models with over 35,000 features to predict the tumor-specific total mRNA expression (TmS) across the two lung cancer types.
- Performed survival analysis on TCGA datasets to examine time-to-event data, successfully identifying the significant factors influencing patient outcomes, which informed clinical decision-making.
- Conducted Gene Set Enrichment Analysis (GSEA) with over 30,000 genes to validate the results, identifying the significantly over-expressed genes and pathways associated with cancer, contributing to the understanding of disease mechanisms.

**Project Intern - Indian Institute of Science**  
_Jan '23 - Jun '23_  
Bengaluru, India  

- Spearheaded rare variant-specific genome wide association studies (GWAS) utilizing WGS data in a cohort of over 2000 individuals.
- Developed and optimized an extensive parallelized computational pipeline to detect disease associations with rare variants and identify causal variants, and interpretation of the GWAS results using statistical plots.
- Developed an automated computational pipeline to run quality control measures and downstream genomic data analyses.

**Research Intern - Leucine Rich Bio Pvt Ptd**  
_Jun '22 - Dec '22_  
Bengaluru, India  

- Curated and managed a comprehensive database highlighting key insights on the gut microbiome and its association with neurodegenerative diseases, specifically Parkinson's disease (PD), facilitating data interpretation.
- Explored the implications of gut microbiome on PD through Bioinformatics data analysis on Galaxy.

---

## Projects

1. **Alzheimer's Disease classifier from fMRI data** | **[GitHub](https://github.com/surakshavinod/Alzheimer-Disease-Classifier)**  

Leveraged machine learning approaches to accurately classify and predict Alzheimer's Disease patients, Mild Cognitive Impairment patients, and Healthy Controls using 60 fMRI scans from the ADNI database, achieving a prediction accuracy of 78%.


2. **CityScout - Identifying Safe Cities Based on Crime Statistics** | **[GitHub](https://github.com/surakshavinod/CityScout)**  

Lauched a web-based tool to identify safe U.S. cities for various use cases (e.g., girls' trips, family relocation, vacation with children, etc.) and customizable filters tailored to specific demographics (e.g., resident, gender, children, etc.) by analyzing 2TB of FBI crime data using PySpark on Google Cloud Platform and Tableau for data visualization.


3. **Comparison of ML models to predict absorption properties** | **[GitHub](https://github.com/surakshavinod/Model-Comparison-for-Absorption-Profile-Prediction)**  

Applied and compared four machine learning models to predict human plasma protein binding rate (PPBR), bioavailability (BIO) and parallel artificial membrane permeability assay (PAMPA) aiming to contribute to the advancement of predictive modeling in drug discovery. Utilized at least 11 features per property and the PyTDC data to build models, achieving a model accuracy of 71%.


---

## Manuscripts and Presentations

1. Paul, A., Ji, S., Lal, J.C., Fong, C., **Vinod, S.**, Hu, H., et al. Machine learning models predict tumor-specific total mRNA expression associated with clinical outcomes across 14 cancer types [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; _Cancer Res 2025_;85(8_Suppl_1):Abstract nr 1095.

2. **Vinod, S.**, Rajesh, S., KN, S., Karunakar, P. (2024). Identifying anti-diabetic potential of alpha-amylase inhibitors in _Ocimum basilicum_ using In vitro and In silico approaches. _Suranaree Journal of Science and Technology, 30_(6), 030159(1-10). https://doi.org/10.55766/sujst-2023-06-e01323

3. **Vinod, S.**, Karunakar, P. (Poster) Identification of potential inhibitor against Nipah virus using In silico approach. _I-SENTIENT_, School of Sciences, Jain (Deemed-to-be) University. September 2022.

4. **Vinod, S.**, Rajesh, S., Karunakar, P., KN, S. Evaluating the anti-diabetic activity of alpha-amylase inhibitors in _Cuminum cyminum_ using In vitro and In silico approach. _International Journal of Computational Biology and Drug Design_. Under review.

---

## Honors and Awards
  
- **Computational Biology Faculty Research Award (x2)**  
*Georgia Institute of Technology*, 2024  
  
- **Prof. M. R. Doreswamy Merit Scholarship (x5)**  
*PES University*, 2020‚Äì2023  
  
- **Best Paper Award**  
*I-SENTIENT Conference, School of Sciences at Jain (Deembed-to-be University)*, 2022  
  
- **1st Runner-up ‚Äì Nutrathon**  
*PES University*, 2022  
  
- **Millennium Fellowship**  
*United Nations Academic Impact & Millennium Campus Network*, 2021  
  
- **2nd Runner-up ‚Äì Druggathon**  
*PES University*, 2021  
  
- **Distinction Award (x2)**  
*PES University*, 2020‚Äì2023  

---

## Skills

 - **Programming Languages:** Python, R, Bash, SQL
 - **Bioinformatics Software:** BLAST, Galaxy, AlphaFold, AutoDock Vina, Desmond, GSEA, PLINK, VCF/BCF Tools, Annovar, SwissADME, OpenBabel, LigPlot+, CellDesigner
 - **Workflow Management and Containers:** Nextflow, Docker, conda/miniconda
 - **Version Control:** Git
 - **Libraries:** numpy, pandas, scikit-learn, Matplotlib, DESeq2, dplyr, tidyr, Seurat, ggplot2, survival, survminer, TCGAbiolinks, seaborn, shiny
 - **Other:** HPC, AWS, Microsoft Azure, OpenRefine, PySpark
 - **Languages:** English, Hindi, Kannada, French (Beginner), German (Beginner)

---

## References
_Available on request._
